index.html

```diff
@@ -31,5 +31,124 @@ <h1>Hey there. I’m Santiago.</h1>
             </ul>
         </div>
     </section>
+    <section class="works">
+        <div class="works-title">
+            <div><img></div>
+            <h2>Works</h2>
+            <div><img></div>
+        </div>
+        <div class="works-card">
+            <div>
+                <h3>Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <div><p>See Project</p></div>
+            </div>
+        </div>
+        <div class="works-card">
+            <div>
+                <h3>Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <div><p>See Project</p></div>
+            </div>
+        </div>
+        <div class="works-card">
+            <div>
+                <h3>Multi-Post Stories Gain+Glory</h3>
+                <ul>
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <div><p>See Project</p></div>
```

Comment:
Kindly use `<a>` tag `<button>` tag for the button because it is going to link us to another page

please repeat this for all buttons

Also, make sure to implement hover and enable effects on the buttons based on [figma designs](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)